mythril/interfaces/cli.py

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python3
-# -*- coding: UTF-8 -*-
+# -*- coding: utf-8 -*-
```

Comment:
About this... Emacs 25.2.2 reports this for UTF-*

```
Warning (mule): Invalid coding system ‘UTF-8’ is specified
for the current buffer/file by the :coding tag.
It is highly recommended to fix it before writing to a file.
```
I am assuming this line is here _only_ for GNU Emacs.